scripts/fix_lint.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Copyright 2018-present Kensho Technologies, LLC.
+
+# Assert script is running inside pipenv shell
+if [[ "$VIRTUAL_ENV" == "" ]]
+then
+    echo "Please run pipenv shell first"
+    exit 1
+fi
```

Comment:
Consider reusing the approach we use elsewhere:
```
# Get into the graphql-compiler virtualenv. "pipenv shell" returns exit code 1 if already there,
# which doesn't matter to us so we avoid crashing the script by adding "|| true".
pipenv shell || true
```

Reply:
Doesn't work when outside pipenv shell, and prints confusing messages when already in pipenv shell